src/main/java/edu/harvard/iq/dataverse/MailServiceBean.java

```diff
@@ -467,14 +467,11 @@ public String getMessageTextBasedOnNotification(UserNotification userNotificatio
                 version =  (DatasetVersion) targetObject;
                 pattern = BundleUtil.getStringFromBundle("notification.email.wasReturnedByReviewer");
                 String optionalReturnReason = "";
-                /*
-                FIXME
-                Setting up to add single comment when design completed
-                optionalReturnReason = ".";
+
                 if (comment != null && !comment.isEmpty()) {
-                    optionalReturnReason = ".\n\n" + BundleUtil.getStringFromBundle("wasReturnedReason") + "\n\n" + comment;
+                    optionalReturnReason = ".\n\n" + MessageFormat.format(BundleUtil.getStringFromBundle("notification.email.wasReturnedByReviewerReason"), comment);
```

Comment:
Some comment from @kmika11:

'looks great! Is the "You may contact us for support at dataverse@localhost" note set at the installation level? This is HDV specific, but it is a quite a pain for everyone to have researchers contacting us at support@dataverse.harvard looking for more info about why their dataset was returned and we have to refer them back to the collection administrator...

I think a "You may contact the collection administrator for more information" would be fine, but it would be really nice to have it dynamically generated based on the collection: "like "You may contact the editorial team at AJPS for more information" or even better, make it possible for the collection admin to write the note they want to have sent: "Please contact [editor@journal.com](mailto:editor@journal.com) for more information."'

For context, here's a screenshot:

![Screenshot 2023-11-29 at 9 34 55 AM](https://github.com/IQSS/dataverse/assets/21006/52dfb11f-18b5-4af5-9c7b-3734a1ff9b85)

Reply:
1. For now I added `You may contact the collection administrator for more information : contact@collection.com` message. 
It can be changed to `You may contact the editorial team at <emails list> for more information` 
2. Customisation of the written note from administrator can be expressed in a new issue as it's a bit complicated for the scope of this PR. 

@pdurbin can you check this with @kmika11 ?
@DS-INRA is fine with the last development

Reply:
Thanks @luddaniel - I think option 1 is great. Definitely understand that anything else is out of scope. Thank you!